pkgs/development/compilers/llvm/git/clang/default.nix

```diff
@@ -63,6 +63,10 @@ let
       (cd tools && ln -s ../../clang-tools-extra extra)
     '' + lib.optionalString stdenv.hostPlatform.isMusl ''
       sed -i -e 's/lgcc_s/lgcc_eh/' lib/Driver/ToolChains/*.cpp
+    '' + lib.optionalString stdenv.hostPlatform.isDarwin ''
+      # Make sure clang passes the correct location of libLTO to ld64
+      substituteInPlace clang/lib/Driver/ToolChains/Darwin.cpp \
+        --replace-fail 'StringRef P = llvm::sys::path::parent_path(D.Dir);' 'StringRef P = "${lib.getLib libllvm}";'
```

Comment:
Shouldn't this be unconditional, so that Linux users catch it if the replacement starts failing?

Reply:
Sure, I can make it unconditional.

Reply:
It’s unconditional now. I also moved it to the right phase (except for clang 16, which was already in `postPatch`).